fulp_modules/main_features/mentorhelp/mentorwho.dm

```diff
@@ -21,6 +21,8 @@
 		for(var/client/C in GLOB.mentors)
 			if(C.is_afk())
 				continue
-			if(C.mentor_datum && !check_rights_for(C, R_ADMIN,0))
+			if(mentor_datum?.is_contributor && !check_rights_for(C, R_ADMIN,0))
+				msg += "\t[C] is a Contributor\n"
+			else if(C.mentor_datum && !check_rights_for(C, R_ADMIN,0))
```

Comment:
copypaste code wtf dies of cringe

check_rights_for should probably be its own if statement rather than having it be there twice